src/asset_folder_importer/premiere_get_referenced_media/processor.py

```diff
@@ -224,9 +225,19 @@ def process_premiere_project(filepath, raven_client, vs_pathmap=None, db=None, c
                 lg.error("File {0} could not be found in either Vidispine or the asset importer database".format(server_path))
             except UnicodeEncodeError:
                 lg.error("File {0} could not be found in either Vidispine or the asset importer database".format(server_path.encode('utf-8')))
-            if "Internet Downloads" in filepath:
+            if ("Internet Downloads" in filepath) or ("Editorial Users" in filepath):
```

Comment:
would you mind making this a configuration entry? so in the config there is a list of paths which is considered invalid

Reply:
Will do.